frontend/javascripts/oxalis/model/reducers/skeletontracing_reducer.js

```diff
@@ -334,6 +334,22 @@ function SkeletonTracingReducer(state: OxalisState, action: Action): OxalisState
             )
             .getOrElse(state);
         }
+        case "RESET_SKELETON_TRACING": {
+          const newTree = createTree(state, Date.now()).get();
```

Comment:
Unfortunately this fails if the current tree is part of a group, because the `createTree` function will assign that same groupId to the newly created tree. Afterwards, you're deleting all tree groups though, which is why the `newTree` is becoming a "ghost" tree, not showing up in the tree tab.
I suppose you could add a parameter to createTree to not reuse the groupId to circumvent this :)

Reply:
Thanks for paying so much attention and finding this bug. And also for the suggested way of fixing this :+1: